demisto_sdk/commands/common/GitContentConfig.py

```diff
@@ -32,7 +32,7 @@ class GitContentConfig:
         CONTENT_GITHUB_MASTER_LINK: Link to the content git repository's master branch
         Credentials: Credentials to the git.
     """
-    BASE_RAW_GITHUB_LINK = r'https://raw.githubusercontent.com/'
+    BASE_RAW_GITHUB_LINK = r'https://raw.{}.com/'
```

Comment:
note this does not support custom domains such as `gitfoo.local`

Reply:
this is a typo, the `com` is there by mistake. fixed